ipn/ipnlocal/local.go

```diff
@@ -786,9 +785,10 @@ func (b *LocalBackend) Start(opts ipn.Options) error {
 	b.send(ipn.Notify{BackendLogID: &blid})
 	b.send(ipn.Notify{Prefs: prefs})
 
-	if wantRunning {
```

Comment:
This would regress the fix to #1573. You can verify with TS_DEBUG_PANIC_MACHINE_KEY=1 set true while starting a fresh tailscaled.

Reply:
I added a test to make sure this can't be broken. Rebase before you run tests again.